applications/dashboard/js/dashboard.js

```diff
@@ -625,6 +625,7 @@ var DashboardModal = (function() {
             if (this.settings.modalType !== 'noheader' && $title.length !== 0) {
                 title = $title.html();
                 $title.remove();
```

Comment:
We need to remove the title if the header block doesn't exist. I updated the block to make this clearer.